Assets/Scripts/GameboardObjects/LineController.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LineController : MonoBehaviour
```

Comment:
This should be a singleton so we can call it with LineController.Instance.SetupLine(...)